ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.test.js

```diff
@@ -54,27 +62,31 @@ describe('Blockaid Banner Alert', () => {
     );
 
     expect(warningBannerAlert).toBeInTheDocument();
-    expect(warningBannerAlert).toMatchSnapshot();
```

Comment:
I had to remove all the snapshots in this file because the URL generated for reportURL is different on local and CI, therefore cause tests to fail on CI but passing locally.

I tried to mock encodeURIComponent using the below 

```
jest
  .spyOn(global, 'encodeURIComponent')
  .mockImplementation((_str) => 'mockedEncodedString');
```

Once again this worked on local and you can see  `mockedEncodedString` in the snapshots, but not working on CI. It was not mocked on CI and CI still returns the encoded string.